README.md

```diff
@@ -80,7 +80,7 @@ class ExampleServiceClient
       # the store you want to use to save the circuit state so it can be
       # tracked, this needs to be Moneta compatible, and support increment
       # this overrides what is set in the global configuration
-      cache: Circuitbox::MemoryStore.new,
+      circuit_store: Circuitbox::MemoryStore.new,
```

Comment:
can we capture this in changelog since it's a breaking change?

Reply:
Yep, I'll do that when I work on the documentation.